### ISSUE TITLE
Build something crazy with Scratch

### ISSUE THREAD

**Issue by bouwe77** [curiouscoderscommunity/curious-coders#34]
https://scratch.mit.edu/

**Comment by RichStone** [comment]
Funny, I have a book about Scratch (in German) and I have it on my list to skim through it for educational purposes. Where does your desire come from? Kids?

**Comment by bouwe77** [comment]
Not directly from kids, but that would be nice as well. I did a workshop at a school once, for 10-12 year olds, where we built a pacman-like game together from some components I created in advance, was really nice!

Not sure what to build with it, maybe an episode where we build something totally ridiculous or crazy with it?

Or instead, come up with an app that (according to us) sums up the most important things we think children should learn, so they get a glimpse of what it's like to be a good dev?

Or, yeah don't know... 🤔 It just fits in our approach to try new things, being curious, and something completely different...

**Comment by RichStone** [comment]
yeah, like all the ideas (whatever totally ridiculous or crazy might mean 😁)
